docs/01-app/02-guides/css-in-js.mdx

```diff
@@ -32,7 +32,7 @@ The following are currently working on support:
 
 > **Good to know**: We're testing out different CSS-in-JS libraries and we'll be adding more examples for libraries that support React 18 features and/or the `app` directory.
 
-If you want to style Server Components, we recommend using [CSS Modules](/docs/app/building-your-application/styling/css) or other solutions that output CSS files, like PostCSS or [Tailwind CSS](/docs/app/guides/tailwind-css).
+If you want to style Server Components, we recommend using [CSS Modules](/docs/app/getting-started/css) or other solutions that output CSS files, like PostCSS or [Tailwind CSS](/docs/app/guides/tailwind-css).
```

Comment:
We might want to add more nuance here. It's specifically runtime CSS-in-JS that is difficult. Maybe we just remove the recommendation.